src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,89 @@
+package Duke;
+
+import java.util.Scanner;
+public class Duke {
+    public static Scanner sc;
+    public static TaskManager taskManager = new TaskManager();
+    public static InputParser parser = new InputParser();
+
+    public static void executeUserInput() throws UnknownCommandException {
+        String command = parser.getCommand();
+        String[] parameters = parser.getTaskParameters();
+
+        switch (command) {
+            case ("list"):
+                taskManager.listAllTask();
+                break;
+            case ("mark"):
+                taskManager.setTask(Integer.parseInt(parameters[0]) - 1, true);
+                break;
+            case ("unmark"):
+                taskManager.setTask(Integer.parseInt(parameters[0]) - 1, false);
+                break;
+            case ("todo"):
+                taskManager.addTodo(parameters[0]);
+                break;
+            case ("deadline"):
+                taskManager.addDeadline(parameters[0], parameters[1]);
+                break;
+            case ("event"):
+                taskManager.addEvent(parameters[0], parameters[1]);
+                break;
+            default:
+                throw new UnknownCommandException();
+        }
+
+    }
+
+    public static void greetUser() {
+
+        final String LOGO = "\n"
```

Comment:
Consider moving the `final` variable to make it a global variable in this class.